rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -923,6 +923,9 @@ bool VKGSRender::load_program()
 		}
 	}
 
+	properties.cs.attachmentCount = 1;
```

Comment:
Confirmed set attachmentCount to m_draw_buffers_count fixed the debug error . I will merge this one .